src/kaleidoscope/device/dygma/raise/attiny_firmware.h

```diff
@@ -0,0 +1,38 @@
+/* -*- mode: c++ -*-
```

Comment:
I'm not happy about including a pre-generated file here, and will figure something better out for the final version. I did want to show the shape of it, which is why it is included.

One possible solution would be to change the flasher API to take the firmware settings as an arg, instead of as a property. That way we could easily replace them in the firmware sketch. In this case, we could ship with dummy defaults, and have the flasher notice that (firmware_length 0, pages 0, etc), and bail out early in that case.

Shipping this in the firmware sketch is easier, because that's going to be a separate, vendor-controlled repo, thus, can be updated at will, without Kaleidoscope maintainers being a bottle-neck. We also don't pollute the repo with generated content.